### PR TITLE
TL translations - 1,341 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
Here's another set of translations I've covered so far. Feel free to review my work before it's merged.

### CODE REVIEW

File: book/03-git-branching/sections/nutshell.asc [hunk 1]
```diff
@@ -1,81 +1,60 @@
 [[_git_branches_overview]]
-=== Branches in a Nutshell
+=== Mga Branch sa Maikling Salita
 
-To really understand the way Git does branching, we need to take a step back and examine how Git stores its data.
+Upang talagang maintindihan ang paraan ng Git sa paggawa ng pag-branch, kailangan nating umatras at suriin kung paano nag-iimbak ng data ang Git.
 
-As you may remember from <<_getting_started#_getting_started>>, Git doesn't store data as a series of changesets or differences, but instead as a series of snapshots.
+Kung iyong iyong natatandaan mula sa <<_getting_started#_getting_started>>, ang Git ay hindi nag-iimbak ng data bilang isang serye ng mga changeset o mga kaibahan, ngunit sa halip ay isang serye ng mga snapshot.
```
Comment (cherryloua) [inline book/03-git-branching/sections/nutshell.asc]: I think this is a typo " Kung iyong iyong natatandaan"

File: book/03-git-branching/sections/nutshell.asc [hunk 2]
```diff
@@ -1,81 +1,60 @@
...
 
-To visualize this, let's assume that you have a directory containing three files, and you stage them all and commit.
-Staging the files computes a checksum for each one (the SHA-1 hash we mentioned in <<_getting_started#_getting_started>>), stores that version of the file in the Git repository (Git refers to them as blobs), and adds that checksum to the staging area:
+Upang maisalarawan ito, ipagpalagay natin na ikaw ay may isang direktoryong naglalaman ng tatlong mga file, at na-stage mo lahat ang mga ito at na-commit. Ang pag-stage ng mga file ay nagkakalkula ng isang checksum para sa bawat isa (ang SHA-1 na hash na nabanggit namin sa <<_getting_started#_getting_started>>), nag-iimbak sa bersyong iyon ng file sa Git na repositoryo (tinutukoy ng Git ang mga ito bilang mga blob), at nagdaragdag ng checksum na iyon sa staging na lawak:
```
Comment (cherryloua) [inline book/03-git-branching/sections/nutshell.asc:11]: I don't know if I'm right but is it better if we translate "staging area" to "staging na lugar"?

File: book/03-git-branching/sections/nutshell.asc (hunk 1 above)
Comment (jjrang11) [inline book/03-git-branching/sections/nutshell.asc]: Please remove the duplicate `iyong`

File: book/03-git-branching/sections/nutshell.asc [hunk 3]
```diff
@@ -1,81 +1,60 @@
...
 
-When you make a commit, Git stores a commit object that contains a pointer to the snapshot of the content you staged.
-This object also contains the author's name and email, the message that you typed, and pointers to the commit or commits that directly came before this commit (its parent or parents): zero parents for the initial commit, one parent for a normal commit, and multiple parents for a commit that results from a merge of two or more branches.
+Kapag ikaw ay gumawa ng isang commit, ang Git ay nag-iimbak ng isang commit na object na naglalaman ng isang pointer sa snapshot ng nilalaman na iyong na-stage. Ang object na ito ay naglalaman din ng pangalan ng may-akda at email, ang mensahe na iyong na-type, at mga pointer sa commit o mga commit na direktang dumating bago ang commit na ito (magulang nito o mga magulang): zero na mga magulang para sa paunang commit, isang magulang para sa isang normal na commit, at maramihang mga magulang para sa isang commit na nagreresulta mula sa isang pag-merge ng dalawa o higit pang mga branch.
```
Comment (jjrang11) [inline book/03-git-branching/sections/nutshell.asc]: Please change `Ang object na ito ay naglalaman din ng pangalan ng may-akda at email` to `Ang object na ito ay naglalaman din ng pangalan at email ng may-akda`

**Review by cherryloua — COMMENTED** [review]
I only saw few or a single issue in this translation. Please, check them out.

**Review by jjrang11 — COMMENTED** [review]
Hey @josteinfrancois i've suggested a few changes for your pull request

File: book/03-git-branching/sections/nutshell.asc (hunk 1 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/nutshell.asc]: Thank you for pointing that out. It was indeed a typo. Changing it now.

File: book/03-git-branching/sections/nutshell.asc (hunk 2 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/nutshell.asc:11]: I think I will have to keep this one. The word 'lawak' makes more sense than 'lugar' in this context. Nonetheless, thank you for pointing it out.

File: book/03-git-branching/sections/nutshell.asc (hunk 1 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/nutshell.asc]: Changing it now.

File: book/03-git-branching/sections/nutshell.asc (hunk 3 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/nutshell.asc]: Thank you for pointing that out. Will update now.

**Pull request merged**